src/components/navigation.jsx

```diff
@@ -0,0 +1,29 @@
+import { NavLink } from 'react-router-dom';
+import logo from '../assets/logo.png';
+import '../styles/navigation.css';
+
+function NavBar() {
+  return (
+    <nav>
+      <div className="parentcontainer300">
+        <div className="container300">
+          <img className="logo300" src={logo} alt="Space Traveler" />
```

Comment:
- [ ] From the missionList.jsx you have not import the file fromthe components kindly fix it to make your page render and visible it display not on the screen when i clone it severally ,please kindly fix it.